### PR TITLE
Update Korean translation

### PR DESCRIPTION
**Pull request by darkcircle**
This is re-PR of #19082 due to my mistake. 

Thank y'all for effort and helping each other.

### CODE REVIEW

**Comment by yuwata** [comment]
Thanks. (Next time, it is ok to use force-push rather than opening new PR.)

File: po/ko.po [hunk 1]
```diff
@@ -434,15 +437,15 @@ msgstr ""
...
 
 #: src/login/org.freedesktop.login1.policy:290
 msgid ""
 "Authentication is required to suspend the system while an application is "
 "inhibiting this."
 msgstr ""
-"프로그램이 절전 상태 진입을 못하게 요청할 때 시스템을 절전 상태로 놓으려면 인"
-"증이 필요합니다."
+"프로그램에서 대기 모드 진입을 막을 때 시스템을 대기 상태로 설정하려면 인증이 "
+"필요합니다."
```
Comment (keszybz) [inline po/ko.po:445]: Hmm, translate.google.com tells me this changes "sleep state" to "standby". Is the new form something that is used commonly, consistent with the strings used e.g. in gnome-shell?

**Comment by keszybz** [comment]
We merged another PR from weblate in the meantime. @darkcircle please rebase.

File: po/ko.po (hunk 1 above)
Comment (darkcircle) [reply po/ko.po:445]: Yes, that's right. refer: https://gitlab.gnome.org/GNOME/gnome-tweaks/-/blob/5a43242bca3963433cefecc492fab9b772454d47/po/ko.po#L309

**Comment by keszybz** [comment]
We like to keep our history pristine, i.e. no back-merges and such. I now rebased this locally. The only change is the removal of a blank line from the end of the file.

File: po/ko.po [hunk 2]
```diff
@@ -72,24 +72,24 @@ msgstr "홈 영역 생성"
 
 #: src/home/org.freedesktop.home1.policy:14
 msgid "Authentication is required to create a user's home area."
-msgstr "사용자 홈 영역을 생성하기 위해서는 인증이 필요합니다."
+msgstr "사용자 홈 영역을 만드려면 인증이 필요합니다."
```
Comment (dongsupark) [inline po/ko.po]: s/만드려면/만들려면/ ?

**Review by dongsupark — COMMENTED** [review]
Mostly looks good to me.

A comment below:

File: po/ko.po (hunk 2 above)
Comment (darkcircle) [reply po/ko.po]: @dongsupark Your suggestion is better. but this merging action is in progress... We would better to wait until it finished. You can catch a chance to fix this problem. :)

**Comment by darkcircle** [comment]
@keszybz Is there something problem? many tests seem to be failed due to conflicts caused by my pull request

**Comment by keszybz** [comment]
> @keszybz Is there something problem? many tests seem to be failed due to conflicts caused by my pull request

No, the tests failed because of #19154.

> Your suggestion is better. but this merging action is in progress... We would better to wait until it finished. You can catch a chance to fix this problem. :)

I you can, please update this PR with this suggestion. A separate patch is OK.

**Comment by darkcircle** [comment]
> > @keszybz Is there something problem? many tests seem to be failed due to conflicts caused by my pull request
> 
> No, the tests failed because of #19154.
> 
> > Your suggestion is better. but this merging action is in progress... We would better to wait until it finished. You can catch a chance to fix this problem. :)
> 
> I you can, please update this PR with this suggestion. A separate patch is OK.

@keszybz Thank you for letting me know that. 

@dongsupark is a great reviewer. I really respect him who has hawk's eye to catch error.

**Comment by poettering** [comment]
hmm, this needs a rebase again. any chance you can rebase? if @dongsupark likes this then, let's merge it then

**Comment by darkcircle** [comment]
It's ok. all is done.

File: po/ko.po [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 #
 # Korean translation for the systemd.
-# Seong-ho Cho <shcho@gnome.org>, 2015.
+# Seong-ho Cho <shcho@gnome.org>, 2015, 2021.
 # Dongsu Park <dongsu@endocode.com>, 2015.
```
Comment (darkcircle) [inline po/ko.po:5]: I will add 2021 onto this line, later. :) Thank you, Dongsu Park!

Comment (dongsupark) [reply po/ko.po:5]: Thanks for pointing it out.
I have realized, my email address there also needs an update. Will do that later. :-)

**Review by dongsupark — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**